### PR TITLE
Fix username display for BCeID users

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/

*Description of changes:*
The username for BCeID user is not loaded properly (showing as 'undefined')
image

Problem resolved. See this screenshot taken from localhost
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-105-b6nse7vf.web.app

File: src/composables/useBcrosAuth.ts [hunk 1]
```diff
@@ -44,9 +44,12 @@ export const useBcrosAuth = () => {
           // successfully initialized so setup other pieces
           keycloak.syncSessionStorage()
           keycloak.scheduleRefreshToken()
+
           // set user info
-          console.info('Setting user name...')
-          await account.setUserName()
+          // NB. user name will be set automatically; calculated from the kcUser initialized in initKeyCloak function
+          // console.info('Setting user name...')
+          // await account.setUserName()
```
Comment (severinbeauvais) [inline src/composables/useBcrosAuth.ts:51]: Please delete commented-out code some time.

Comments explaining why something is the way it is are good 👍

Comment (patrickpeinanw) [reply src/composables/useBcrosAuth.ts:51]: Good point. I will delete these lines in my next PR.

**Pull request merged**